demos/browser/app/meetingReadinessChecker/meetingReadinessChecker.ts

```diff
@@ -91,24 +96,33 @@ export class DemoMeetingApp {
     this.defaultBrowserBehaviour = new DefaultBrowserBehavior();
     this.meeting = `READINESS_CHECKER-${uuidv4()}`;
     this.name = `READINESS_CHECKER${uuidv4()}`;
-    this.region = (document.getElementById('inputRegion') as HTMLInputElement).value;
-    //start meeting
+    // Initialize logger and device controller to populate device list
     new AsyncScheduler().start(
       async (): Promise<void> => {
-        let chimeMeetingId = '';
-
-        try {
-          chimeMeetingId = await this.authenticate();
-          this.log(`chimeMeetingId: ${chimeMeetingId}`);
-          const button = document.getElementById("authenticate") as HTMLButtonElement;
-          button.disabled = false;
-        } catch (error) {
-          return error;
-        }
+        await this.initializeLoggerAndDeviceController();
       }
     );
   }
 
+  async startMeetingAndInitializeMeetingReadinessChecker(): Promise<void> {
+    //start meeting
+    let chimeMeetingId = '';
+    try {
+      this.region = (document.getElementById('inputRegion') as HTMLInputElement).value;
+      chimeMeetingId = await this.authenticate();
+      this.log(`chimeMeetingId: ${chimeMeetingId}`);
+    } catch (error) {
+      return error;
```

Comment:
Should we have a failed meeting page to display the error like in meeting demo? Something like this:
```
          } catch (error) {
            (document.getElementById(
              'failed-meeting'
            ) as HTMLDivElement).innerText = `Meeting ID: ${this.meeting}`;
            (document.getElementById('failed-meeting-error') as HTMLDivElement).innerText =
              error.message;
            this.switchToFlow('flow-failed-meeting');
            return;
          }
```

Reply:
Yep that's a good point. I will update this.